### DISCUSSION TITLE
Error Download videos Nutror

### DISCUSSION THREAD

**Discussion by tpcorradi** [aclap-dev/video-downloadhelper#1048] [Help and Q&A]

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

First, you should paste in ALL of your information from the `Copy information to clipboard` button.  Second, you should search this forum for other discussions of that web site.  There are several such.

**Comment by Mattmjs** [comment]
It may work in the beta extension for Firefox, try it out and see what happens. Then come back with your results.

**Comment by tpcorradi** [comment]


**Reply by Mattmjs** [reply]
I was hoping the beta would work. Can you provide more information by clicking the 3 dots next to download.
Look down to Details, clicking this will open a page with the information about the video that was detected.
Copy this as text ( not an image ) and paste it here. This will help more to understand the problem.

**Comment by tpcorradi** [comment]


**Reply by Mattmjs** [reply]
I don't know why it would fail, it works directly in ffmpeg. It downloaded all the way to the end. This looks to be possible in VDH.
@paulrouget  might be able to figure this & get it working.

**Reply by GrampaWildWilly** [reply]
Whenever you try something like this & manage to get it to work in ffmpeg when VDH fails (which it did for me as well), you really ought to post your ffmpeg log for @paulrouget Paul's benefit.

[Nutror Test mp4 Log.txt](https://github.com/user-attachments/files/16399063/Nutror.Test.mp4.Log.txt)

It worked in ffmpeg for me, too.  Do note, however, that the ffmpeg run did not go smoothly.  There were a lot of this:

Cannot reuse HTTP connection for different host

There were some knock-on errors associated with those.  There was also a modest level of corrupt packets.  I'm using the `-fflags +discardcorrupt` parameter on ffmpeg so such packets don't automatically terminate the download.  I hope the CoApp is using this parameter.  I suspect all this error recovery activity was slowing down the download to the rather mediocre level you can see reported in that log.

Add this one to #1044 which also was a case that looked like VDH should have handled it but didn't.  That one also involved a corrupt video that VDH was able to download despite the errors.  Are we seeing a pattern?

**Comment by tpcorradi** [comment]


**Reply by GrampaWildWilly** [reply]
It's looking very much like there is a bug in VDH here.  Our only recourse on that front is to wait until Paul fixes this.

Changing the subject now.  I want to emphasize, I'm starting to discuss a separate topic.  There is a tool called ffmpeg that you can use INSTEAD of VDH.  That's what I used to perform the download I did above.  It so happens that VDH also uses ffmpeg.  In fact, I used the same ffmpeg VDH uses.  But for some reason, VDH fails but using ffmpeg directly yourself works.  You will need to learn about ffmpeg.  Here's some educational resources:

https://groups.google.com/g/video-downloadhelper-q-and-a/c/oYDFx367MSY
https://groups.google.com/g/video-downloadhelper-q-and-a/c/sNfTCMYfiTU

**Comment by paulrouget** [comment]
@Mattmjs or @tpcorradi  I need a public URL that fails.

@tpcorradi I also need you to do this: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F and then post the result here.

**Reply by Mattmjs** [reply]
I don't think there are public urls , there is nothing on the nutror homepage. Pasting the m3u8 link in a new tab does the
job of detecting the video but fails once the download starts.

**Reply by Mattmjs** [reply]
I'm not a member of nutror, @tpcorradi  is a member, that's why I got them to share the details you see which you've edited.

**Reply by paulrouget** [reply]
This is fixed in v9.1.0.39 (firefox beta - should be live in ~10 minutes).

**Comment by tpcorradi** [comment]
hello ! 
again the extension stopped downloading on the platform.

Would anyone have an idea of ​​the solution?

<html><body>

Tab | 7 (incognito: false)
-- | --
https://app.nutror.com/curso/cdf576ca13573/aula/203480 with 0 headers
/content/images/no-thumbnail.png
false
https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/mh/uP/mm/31%2C29/mn/sn-oxunxg8pjvn-wujee%2Csn-bg0s7n7d/ms/au%2Crdu/mv/m/mvi/2/pl/21/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/1152500/vprv/1/go/1/rqh/5/mt/1724718057/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRgIhALF6xQGS4d1uSue9LYDhPmNHHSdJlprh3L-hpJI8UBAbAiEA5T5RsN5hyTkrcKXVJRnFKTzP-CM6zoZpwlvKVocFIQU%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/AGtxev0wRgIhAOiK9Zug1apfE78dd51p-_yfVQJ8GXsnWrvHcnjTnkRTAiEA96CeDXXoarA4Hyb3raS3gU7-kneGTJefITr36brLCMs%3D/file/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/232/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D50723825%3Bdur%3D3175.104%3Bgir%3Dyes%3Bitag%3D136%3Blmt%3D1536675574793363/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRAIgZywB7JB8_HKSKsstF8yt2g8UXraGtVFVY--JgWrsUMwCIH7s_Jf9bUcNO78DzxWMSnVBo-5WhU1rzK_kznEVtQJn/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIhAPiEn3QVfn6TWaoG6kKwMOsJ5uYGIDEynQ0WL1K6Blt9AiB3viVTcUDuSBblZwcz1QN_82rtoobv7eZQ-mRNlsBYfA%3D%3D/playlist/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/234/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/134/sgoap/clen%3D50429943%3Bdur%3D3175.166%3Bgir%3Dyes%3Bitag%3D140%3Blmt%3D1536675410801738/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRgIhAPmZj8yOKDXubScX4QLj2io8mljifJeeQ35tfgqIh9hCAiEAhrO-Ee7mvBJMtU_AC8fRunreSroX_Ee8ld857UBJf68%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIhANYDbDVSMJQKfVwT93YrLlz7aJEVDGjJIDkBbZzYGaAkAiBGQayhCC2DlCE8Lxrmz6OtO2mh3e3O7duGArnKYWLgPw%3D%3D/playlist/index.m3u8
Mp4 b:Hls p:hls 3175.1048810000384s
AAC at unknown
H264 at 743187, fps: 30, q:720, {"width":1280,"height":720}
https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/mh/uP/mm/31%2C29/mn/sn-oxunxg8pjvn-wujee%2Csn-bg0s7n7d/ms/au%2Crdu/mv/m/mvi/2/pl/21/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/1152500/vprv/1/go/1/rqh/5/mt/1724718057/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRgIhALF6xQGS4d1uSue9LYDhPmNHHSdJlprh3L-hpJI8UBAbAiEA5T5RsN5hyTkrcKXVJRnFKTzP-CM6zoZpwlvKVocFIQU%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/AGtxev0wRgIhAOiK9Zug1apfE78dd51p-_yfVQJ8GXsnWrvHcnjTnkRTAiEA96CeDXXoarA4Hyb3raS3gU7-kneGTJefITr36brLCMs%3D/file/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/230/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D19446316%3Bdur%3D3175.104%3Bgir%3Dyes%3Bitag%3D134%3Blmt%3D1536675567973990/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIhANmD9UHp-P7875e6jlmcqigbYDMl84IVTGrvwtjMF8UvAiAlcPGv-H3d-BHOCToN08EbCKh6UwKITDOoFNB7TWPWgg%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIhALd0ExFVX3j-Nb25XtNY7N_8j4Cq-Bya8DBw-IJ1Xxa8AiAs_iTgfwt2GSQAZvVeabo7hxH_2wsMzgQKm19oHxLFdA%3D%3D/playlist/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/234/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/134/sgoap/clen%3D50429943%3Bdur%3D3175.166%3Bgir%3Dyes%3Bitag%3D140%3Blmt%3D1536675410801738/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRgIhAPmZj8yOKDXubScX4QLj2io8mljifJeeQ35tfgqIh9hCAiEAhrO-Ee7mvBJMtU_AC8fRunreSroX_Ee8ld857UBJf68%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIhANYDbDVSMJQKfVwT93YrLlz7aJEVDGjJIDkBbZzYGaAkAiBGQayhCC2DlCE8Lxrmz6OtO2mh3e3O7duGArnKYWLgPw%3D%3D/playlist/index.m3u8
Mp4 b:Hls p:hls 3175.1048810000384s
AAC at unknown
H264 at 338127, fps: 30, q:360, {"width":640,"height":360}
https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/mh/uP/mm/31%2C29/mn/sn-oxunxg8pjvn-wujee%2Csn-bg0s7n7d/ms/au%2Crdu/mv/m/mvi/2/pl/21/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/1152500/vprv/1/go/1/rqh/5/mt/1724718057/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRgIhALF6xQGS4d1uSue9LYDhPmNHHSdJlprh3L-hpJI8UBAbAiEA5T5RsN5hyTkrcKXVJRnFKTzP-CM6zoZpwlvKVocFIQU%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/AGtxev0wRgIhAOiK9Zug1apfE78dd51p-_yfVQJ8GXsnWrvHcnjTnkRTAiEA96CeDXXoarA4Hyb3raS3gU7-kneGTJefITr36brLCMs%3D/file/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/269/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D6167912%3Bdur%3D3175.104%3Bgir%3Dyes%3Bitag%3D160%3Blmt%3D1536675562126134/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRQIgLfQxF_MV8YjRyU0LpVF3E79nfW-Wm19yx139bk1GARgCIQDBZtHC6Qz0pKEky6lQvgREHKrQJkyTkNHSy2RNZkpPcQ%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRgIhAPGE0_34TAwZf2B0cM_oylni3EaObhnEyfmp2MFo3aoSAiEAlbUKBg_0HOwM8sYPwuRHqJjchpmblk8kVztqg2kv3ek%3D/playlist/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/233/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/134/sgoap/clen%3D18883810%3Bdur%3D3175.235%3Bgir%3Dyes%3Bitag%3D139%3Blmt%3D1536675410701261/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRAIgQOpq_0U5E8SCVjpW-vRItRrouD73HADHEkV-Qgojm1cCIEzjBiieXO4ezSetytYmOFWppi6bjVwONomfuFdQPAHn/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRgIhAMVl7i5xbaHfLKqddGt6lxFgDj4OlRgn_iCdaYo4aEKEAiEAtZLTU_H5QxnqsT82pa5Iaa5g9XTo5jvyMDOrUUqeshs%3D/playlist/index.m3u8
Mp4 b:Hls p:hls 3175.1048810000384s
AAC at unknown
H264 at 102412, fps: 30, q:240, {"width":256,"height":144}
https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/source/youtube/requiressl/yes/xpc/EgVo2aDSNQ%3D%3D/playback_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/mh/uP/mm/31%2C29/mn/sn-oxunxg8pjvn-wujee%2Csn-bg0s7n7d/ms/au%2Crdu/mv/m/mvi/2/pl/21/hfr/1/demuxed/1/tts_caps/1/maudio/1/initcwndbps/1152500/vprv/1/go/1/rqh/5/mt/1724718057/fvip/4/nvgoi/1/short_key/1/ncsapi/1/keepalive/yes/dover/13/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cxpc%2Chfr%2Cdemuxed%2Ctts_caps%2Cmaudio%2Cvprv%2Cgo%2Crqh%2Citag%2Cplaylist_type/sig/AJfQdSswRgIhALF6xQGS4d1uSue9LYDhPmNHHSdJlprh3L-hpJI8UBAbAiEA5T5RsN5hyTkrcKXVJRnFKTzP-CM6zoZpwlvKVocFIQU%3D/lsparams/playback_host%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps/lsig/AGtxev0wRgIhAOiK9Zug1apfE78dd51p-_yfVQJ8GXsnWrvHcnjTnkRTAiEA96CeDXXoarA4Hyb3raS3gU7-kneGTJefITr36brLCMs%3D/file/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/605/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/wft/1/sgovp/clen%3D32480953%3Bdur%3D3175.105%3Bgir%3Dyes%3Bitag%3D243%3Blmt%3D1536676244422403/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,wft,sgovp,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRAIgI3x_fpONzZjtpb8kjzU6PmCYL1b0RNGMqVDYHw_L624CIGjhtbumtB_9LV_1HmjKN_tDG7a0s13FijP26OKiuKrJ/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIgD_83pbbbH8AzU-u7tC8G4YcWgdMVxHBAFhERpA3KQsMCIQCGxhCyh96LvdmI3zVZDu64ruS8Km2wT2Q6JhjkzWKoSg%3D%3D/playlist/index.m3u8
https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1724739990/ei/Nh3NZovKK9no-LAP15G9oAI/ip/189.35.238.121/id/d606bacc3f9389a5/itag/234/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/goi/134/sgoap/clen%3D50429943%3Bdur%3D3175.166%3Bgir%3Dyes%3Bitag%3D140%3Blmt%3D1536675410801738/rqh/1/hls_chunk_host/rr2---sn-oxunxg8pjvn-wujee.googlevideo.com/xpc/EgVo2aDSNQ%3D%3D/mh/uP/mm/31,29/mn/sn-oxunxg8pjvn-wujee,sn-bg0s7n7d/ms/au,rdu/mv/m/mvi/2/pl/21/initcwndbps/1152500/vprv/1/playlist_type/DVR/dover/13/txp/5533332/mt/1724718057/fvip/4/short_key/1/keepalive/yes/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,goi,sgoap,rqh,xpc,vprv,playlist_type/sig/AJfQdSswRgIhAPmZj8yOKDXubScX4QLj2io8mljifJeeQ35tfgqIh9hCAiEAhrO-Ee7mvBJMtU_AC8fRunreSroX_Ee8ld857UBJf68%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AGtxev0wRQIhANYDbDVSMJQKfVwT93YrLlz7aJEVDGjJIDkBbZzYGaAkAiBGQayhCC2DlCE8Lxrmz6OtO2mh3e3O7duGArnKYWLgPw%3D%3D/playlist/index.m3u8
Mkv b:Hls p:hls 3175.1048810000384s
AAC at unknown
VP9 at 369739, fps: 30, q:360, {"width":640,"height":360}


</body>
</html>

**Reply by Mattmjs** [reply]
Using one of those google video links I see it working but it is severely throttled to the point that the download could end
prematurely.

**Reply by tpcorradi** [reply]
Yes, and what would be the problem or solution?

**Reply by Mattmjs** [reply]
I've highlighted what the problem is but I don't know what  the solution is. You could screen record as an alternative.

**Reply by tpcorradi** [reply]
Yes, I understand, but for what I need, it is not feasible to record the screen, there are many videos that I need to download.
@paulrouget Can you check if there is a solution?

**Reply by tpcorradi** [reply]
From what I understand, the extension will not be able to support this type of download. I will wait a few more days, and if it doesn’t work, I will look for another one that can perform the procedure. If anyone has an idea or solution, I would appreciate it in advance. @paulrouget @GrampaWildWilly @Mattmjs @jcfrog

**Comment by tpcorradi** [comment]
Update: I discovered that the videos that were not being downloaded were actually deleted, and there were still, for some reason, traces of them on the platform. In another one that I tested, they are being downloaded normally. Before, I removed the extension and installed the BETA version again.
